src/middleware/respondWithData.js

```diff
@@ -1,3 +1,8 @@
 module.exports = function respondWithData(req, res) {
-  res.json(res.locals.questionnaire);
+  var questionnaire = res.locals.questionnaire;
+  var trimmedQuestionnaire = JSON.parse(
```

Comment:
It's probably cleaner to use https://lodash.com/docs/4.17.11#trim

Reply:
Ive tried using `trim` but can only get it to work on the top level. How do I get it to recurse down the object structure?

Reply:
You could recursively `map` down the nested structure but I didn't realise this had to deal with that scenario which probably makes the current solution cleaner. 

Probably best to write a test to indicate that it deals with nested structures.